main.cpp

```diff
@@ -471,12 +467,12 @@ struct AudioApplication
 float AudioApplication::compress(float input, float ratio, float threshold)
 {
     if (input>threshold)
-    {
-        return input * (0.1f*ratio);
-    } else
-    {
-        return input;
-    }
+        {
+            return input * (0.1f * ratio);
+        } else
+            {
+                return input;
+            }
 }
```

Comment:
This indentation is all messed up.   You just needed to put the `else` on its own line to correct the formatting. 
- [ ] revise